gigasecond.md

```diff
@@ -1 +1 @@
-A gigasecond is one billion (10**9) seconds.
+A gigasecond is 10^9 (1,000,000,000) seconds.
```

Comment:
I think it would be a mistake to try to write exponentiation here in a programming language-like syntax. 10^9 is the most straightforward "human" way to write 10-to-the-power-of-9. The superscript 9 I'm a little more open to, but since `1,000,000,000` appears immediately after 10^9, is there really any risk of confusion?